### DISCUSSION TITLE
Subscription only seems to work on node A?

### DISCUSSION THREAD

**Discussion by marcuslindblom** [ravendb/ravendb#13201] [Q&A]
I have a subscription on `@all_docs`. When I create the subscription I don't actively set the MentorNode, I let Raven choose the node. If the subscription ends up on a different node than A, I don't seem to get any data? We use Raven cloud for both dev and production and I experience the same issue on both environments. It seems odd that I must choose to set A but maybe that's how it works?

**Comment by garayx** [comment]
Are you using our cloud solution [https://cloud.ravendb.net/](https://cloud.ravendb.net/) ? if so, please reopen the ticket using the cloud portal [https://cloud.ravendb.net/portal/](https://cloud.ravendb.net/portal/)

**Reply by marcuslindblom** [reply]
Ok 👍🏻
